app/pandas.ipynb

```diff
@@ -391,9 +371,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# use ** operator here"
-   ],
+   "source": "power = s1 **",
```

Comment:
The power operation on the Series 's1' is incomplete. You need to specify the exponent for the power operation.